src/DssSpell.sol

```diff
@@ -39,34 +45,61 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    uint256 constant ZERO_SEVENTY_FIVE_PCT_RATE = 1000000000236936036262880196;
-    uint256 constant ONE_PCT_RATE               = 1000000000315522921573372069;
-    uint256 constant ONE_FIVE_PCT_RATE          = 1000000000472114805215157978;
+    uint256 internal constant MILLION = 10 ** 6;
+    uint256 internal constant BILLION = 10 ** 9;
 
-    uint256 constant MILLION = 10 ** 6;
-    uint256 constant RAY     = 10 ** 27;
+    uint256 internal constant WAD     = 10 ** 18;
 
-    address immutable MCD_SPOT = DssExecLib.spotter();
+    uint256 internal constant PSM_HUNDRED_BASIS_POINTS = 100 * WAD / 10000;
```

Comment:
I'm in favour of calling this `PSM_ONE_HUNDRED_BASIS_POINTS` instead of just 'hundred basis points' for clarity. Either is fine but I would go with one hundred.

Reply:
This was ported from the mainnet spell, so should stay as is as much as possible.
The review should be mostly diffing with the mainnet changes.